Makefile

```diff
@@ -0,0 +1,64 @@
+.PHONY: default
+default: help
+
+.PHONY: help
+help:
+	@echo "make help              Show this help message"
+	@echo "make dev               Run the pattern library application (TODO)"
+	@echo "make build             Build the package"
+	@echo "make lint              Run the code linter(s) and print any warnings"
+	@echo "make checkformatting   Check code formatting"
+	@echo "make format            Automatically format code"
+	@echo "make test              Run the unit tests once"
+	@echo "make sure              Make sure that the formatter, linter, tests, etc all pass"
+	@echo "make clean             Delete development artefacts (cached files, "
+	@echo "                       dependencies, etc)"
+
+.PHONY: dev
+dev: node_modules/.uptodate
+	@echo 'TODO: run local dev server with pattern library'
+
+.PHONY: test
+test: node_modules/.uptodate
+ifdef ARGS
+	yarn test $(ARGS)
+else
+	yarn test
+endif
+
+.PHONY: lint
+lint: node_modules/.uptodate
+	yarn run lint
+	yarn run typecheck
+
+.PHONY: docs
+docs: python
```

Comment:
I think this can be removed

Reply:
Indeed. We need to figure out the optimal tools for documentation for this project but I suspect it will involve a combination of markdown files and generating a static site with the pattern library in it.

Reply:
Whoops. Oversight, thanks.